### ISSUE TITLE
Keyboard control customization

### ISSUE THREAD

**Issue by Gobbel2000** [opensuspect/opensuspect-legacy#167]
For accessibility and customizability there should be a settings submenu for changing all keyboard (and mouse) bindings that are used by the game. Though not an urgent feature, it will help players using different keyboard layout. Especially if the text chat is a critical game component, using a different layout than usual would be very uncomfortable for some players. 

Support for multiple keybinds on one action would be a feature to consider (e.g. allowing movement with WASD and arrow keys). 

The implementation might be linked to and possibly dependent of #99.
